README.md

```diff
@@ -69,15 +69,14 @@ Requirements:
 
 - Ansible >= 2.7.8
 - Jinja >= 2.7
-- pyOpenSSL
```

Comment:
README.md is getting an overhaul in the 4.1 Docs PR.  Maybe drop this file from your PR to prevent merge conflicts?